packages/webdriverjs/tests/example/webdriverjs-example.ts

```diff
@@ -0,0 +1,85 @@
+import { assert } from 'chai';
+import { Builder, WebDriver } from 'selenium-webdriver';
+import * as chrome from 'selenium-webdriver/chrome';
+import AxeBuilder from '@axe-core/webdriverjs';
+
+describe('Deque Webdriverjs example', () => {
+  let driver: WebDriver;
+
+  beforeEach(() => {
+    driver = new Builder()
+      .forBrowser('chrome')
+      .setChromeOptions(new chrome.Options().headless())
+      .build();
+  });
+
+  afterEach(() => {
+    driver.close();
+  });
+
+  describe('Deqeue Homepage', () => {
+    it('should run analysis', async () => {
+      await driver.get('https://deque.com/');
```

Comment:
If this is going to be run in CI, we should not be relying on `deque.com` being up. If the marketing website goes down (maybe they're doing an upgrade, or perhaps Wordpress has exploded), our tests will fail. This will prevent us from hot-fixing critical bugs quickly.

I **strongly** recommend running a local server and testing "fixtures" rather than a live website (which this team does not control).